Babylon/utils/yaml_utils.py

```diff
@@ -18,6 +19,21 @@ def read_yaml_key(yaml_file: pathlib.Path, key: str) -> Optional[object]:
     except:
         return None
 
+def write_yaml_value(yaml_file: pathlib.Path, key: str, value: str) -> None:
+    """
+    This function aloow to update a YAML file values
```

Comment:
I hope you never use this in a loop.
Reading/Writing files can be really slow so you should write the updated dict in one pass.

Reply:
For now i never use this function in a loop and i don't planed to use it in a loop